src/core/renderers/SystemRenderer.js

```diff
@@ -192,23 +185,48 @@ export default class SystemRenderer extends EventEmitter
     }
 
     /**
-     * Resizes the canvas view to the specified width and height
+     * Same as view.width, actual number of pixels in the canvas by horizontal
      *
-     * @param {number} width - the new width of the canvas view
-     * @param {number} height - the new height of the canvas view
+     * @member {number}
+     * @readonly
+     * @default 800
      */
-    resize(width, height)
+    get width()
     {
-        this.width = width * this.resolution;
-        this.height = height * this.resolution;
+        return this.view.width;
+    }
 
-        this.view.width = this.width;
-        this.view.height = this.height;
+    /**
+     * Same as view.height, actual number of pixels in the canvas by vertical
+     *
+     * @member {number}
+     * @readonly
+     * @default 600
+     */
+    get height()
+    {
+        return this.view.height;
+    }
+
+    /**
+     * Resizes the screen and canvas to the specified width and height
+     * Canvas dimensions are multiplied by resolution
+     *
+     * @param {number} screenWidth - the new width of the screen
+     * @param {number} screenHeight - the new height of the screen
+     */
+    resize(screenWidth, screenHeight)
+    {
+        this.screen.width = screenWidth;
+        this.screen.height = screenHeight;
+
+        this.view.width = screenWidth * this.resolution;
+        this.view.height = screenHeight * this.resolution;
```

Comment:
800 x 600 with resolution of 2 means the view number of pixels will be 1600 x 1200 but the css will maintain the 800 x 600 size